### DISCUSSION TITLE
LLM技术和训练问题的微信交流群

### DISCUSSION THREAD

**Discussion by Anna10071** [ymcui/Chinese-LLaMA-Alpaca#612] [General]
如果二维码过期加微信：yydsa0007

![d24e83490b27b7732512cdbd1cb0b64](https://github.com/ymcui/Chinese-LLaMA-Alpaca/assets/90118245/3786009d-0094-433f-b3b6-9de60f0e52c7)

**Comment by xeoshow** [comment]
二维码过期，这个微信号也提示不存在该用户。
